common/core/src/commonMain/kotlin/com/trafi/core/model/DrivingLicence.kt

```diff
@@ -25,13 +25,13 @@ import kotlinx.serialization.internal.CommonEnumSerializer
  */
 @Serializable
 data class DrivingLicence(
-    @SerialName(value = "categories") @Required val categories: kotlin.collections.List<DrivingLicenceCategory>,
-    @SerialName(value = "verificationStatus") @Required val verificationStatus: DrivingLicence.VerificationStatus,
-    @SerialName(value = "licenceNumber") val licenceNumber: kotlin.String? = null,
-    @SerialName(value = "issuedAt") val issuedAt: kotlin.String? = null,
-    @SerialName(value = "expiresAt") val expiresAt: kotlin.String? = null,
-    @SerialName(value = "verificationMessageKey") val verificationMessageKey: kotlin.String? = null,
-    @SerialName(value = "verificationFallbackMessage") val verificationFallbackMessage: kotlin.String? = null
+    @SerialName(value = "categories") @Required var categories: kotlin.collections.List<DrivingLicenceCategory>,
```

Comment:
Not all properties changed in this PR have to be made mutable, e.g. we don't expect SDK users to modify `DrivingLicense`. I'd suggest changing just `Profile` and perhaps some of the `User` properties, if necessary.

Reply:
👍